env.template

```diff
@@ -0,0 +1 @@
+VITE_SOLID_IDENTITY_PROVIDER="http://localhost:3000"

```

Comment:
this will work IF we merge in the PR (#354 ) about the add client modal fix, or we can add a `/` at the end of this line and close that PR and issue without merging.